src/hotspot/share/runtime/globals.hpp

```diff
@@ -550,7 +551,8 @@ const intx ObjectAlignmentInBytes = 8;
           "Dump heap to file after any major stop-the-world GC")            \
                                                                             \
   product(bool, HeapDumpOnOutOfMemoryError, false, MANAGEABLE,              \
-          "Dump heap to file when java.lang.OutOfMemoryError is thrown")    \
+          "Dump heap to file when java.lang.OutOfMemoryError is thrown "    \
+          "from JVM")    \
```

Comment:
It would be better to align `\` to the right.

Reply:
yes, thanks!